site/contents/write_ups/2022/12_01_22-writeup.md

```diff
@@ -0,0 +1,93 @@
+
+## Why is what we do important?
+
+The rules of arithmetic are objective, but the way we use them are not, which leads to inevitable divide between theory in practise. We have to make important decisions about where we spend our money, where we have our conferences and where we put our attention. Data science is just a tool to help us make these decisions.
+
+Maybe the question is 'what are we going to commit to **not** doing' or 'what are we going to call other people out for doing?'. How do we stand up to your superiors, and justify refusing to do something? This is especially difficult in commerical environments. We have found that, as individuals, it is difficult to stick your head above the parapet - you need to have a bit of a community to fall back on. Building a community where you can discuss ethical issues and fall back to is vital for getting more people to act more ethically, even if it is an uphill battle sometimes.
+
+How can we be activists? Making statements to the university etc. Do we want to consider a mission statement? Do we want to encourage development of ethical courses?
+We as a group should think about what kind of impact do we want to have? Recommendations, running courses, 
+
+Data ethics club has the potential to act as a support group. We do not necessarily have all the same views but having data ethics as an important component of our professional practice brings us together. Communities are important to help us improve our own behaviour, holding us accountable whilst supporting us through changes.
+
+A lot of the problems we have discussed over the past year have arrived because there has been nobody present to question people's ethics. Ethics always pertains to structures of power, vested interests and conflicts that pertain to these things. The whole point of data ethics club is to advocate for data science practitioners to practise their craft in a more ethically sound way. Of course, we don't want just a focus on STEM. We need people to be ethical in using data across other fields such as sociology and philosophy.
+
+How do we affect change? We can't change things just from inside universities - they don't work like the rest of the world! We need to consider other outreach options. Ben wrote to Bristol Cable (a community led news source) and they published his article [Opinion: Engineers can’t ignore social responsibility](https://thebristolcable.org/2017/08/social-responsibility-engineers/). This is just one example of how we can speak out about issues in the data ethics field.
+
+## Reaching Out
+
+Spend enough time working on anything in an academic environment, surrounded by likeminded people invested in your topic, it can be neglect making what you do and talk about accessible to non-specialists. Academics in particular suffer with this problem. Addressing some of the problems we have talked about in data ethics club will require a lot more people talking about it than subject specialists.
```

Comment:
```suggestion
Spend enough time working on anything in an academic environment, surrounded by likeminded people invested in your topic, and it can be east to neglect making what you do and talk about accessible to non-specialists. 
Academics in particular suffer with this problem. 
Addressing some of the problems we have talked about in Data Ethics Club will require a lot more people talking about it than subject specialists and many of our resolutions were on the theme of communication. 
```